French Translations

Wawa-byte opened the pull request:
dig_deeper.po and pappers-and-results.po translated into French

Wawa-byte committed:
Add files via upload

Wawa-byte committed:
Add files via upload

Wawa-byte committed:
Delete pappers-and-results.po

ReadyPlayerEmma commented on Localization/fr-FR/dig_deeper.po:
```diff
@@ -0,0 +1,241 @@
+"FAH-Translate-Schema-Version: 1.0\n"
+"URL: https://foldingathome.org/dig-deeper/\n"
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
```
Can you please check that the source for the translation is the latest version. The latest source version for this file was updated on 2020-09-23, and is version 1.2.

meatball133 replied:
If you are wondering what has changed since 1.0: 
https://github.com/FoldingCommunity/Translate/pull/136/commits/1d81098bea42806216b444c3e2b78015ee3aa4e3
https://github.com/FoldingCommunity/Translate/pull/135/commits/16f4abcf964d403dd4efa1f2990aa4b9753cacf3

meatball133 replied:
If you are wondering waht ahs changed since 1.0:
https://github.com/FoldingCommunity/Translate/pull/136/commits/1d81098bea42806216b444c3e2b78015ee3aa4e3
https://github.com/FoldingCommunity/Translate/pull/135/commits/16f4abcf964d403dd4efa1f2990aa4b9753cacf3

meatball133 replied:
Othervise really good job. This fille is massive and is probably the biggest file that will ever exisist on this github. Thanks for the help.

ajmch replied:
Thank you for the heads-up and the help. Actually, the changes are already implemented in the file. We just failed to adapt the source version for the PR, as the translation was ongoing during the changes. It will be done shortly.

Wawa-byte replied:
I have updated the file with the changes requested

ReadyPlayerEmma reviewed (changes requested):
Thank for you the work on translations! Please try to make sure that newly contributed translation PRs are opened with the latest source versions of the translated files.

Wawa-byte committed:
Update dig_deeper.po

Line 6 was updated to match with the source version used.
Line 7 was updated to match with the correct source-last-modified. 
On line 148 changed from "Ne sont-ils pas les numéros PRCG ?" to "Il s'agit des numéros PRCG ?"

ReadyPlayerEmma reviewed (approved):
Thank you!
